authenticity.cpp

```diff
@@ -189,19 +185,19 @@ std::string Authenticity::Assay
         }
 
     // Validate all data files.
-    fs::path original_path(fs::current_path());
-    if(0 != chdir(data_path.string().c_str()))
-        {
-        oss
-            << "Unable to change directory to '"
-            << data_path
-            << "'. Try reinstalling."
-            ;
-        return oss.str();
-        }
     try
         {
-        system_command("md5sum --check --status " + std::string(md5sum_file()));
+        auto const md5sums_path = data_path / md5sum_file();
```

Comment:
Do we really need this temporary variable (also `file_path` below)? I'd just inline it.